### DISCUSSION TITLE
Transferring repo into interTwin-eu

### DISCUSSION THREAD

**Discussion by gray95** [interTwin-eu/repository-template#5] [Q&A] [answered]
Hi all, I'd like to transfer a repo related to the use case T4.1 from WP4 into this org page. I get a permission denied message when I try to initiate the transfer. Any help would be greatly welcomed.

cheers,
Gaurav

**Comment by andrea-manzi** [comment, marked as answer]
Hi Gaurav, 
i just invited you to join the organization, please accept the request and then try again to transfer the repo
thanks
Andrea

**Reply by gray95** [reply]
thanks for the invite andrea, looks like the transfer went through fine.
